addons/im_livechat/models/chatbot_script_question_answer.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, models, fields
+from odoo.osv import expression
+
+
+class ChatbotScriptQuestionAnswer(models.Model):
+    _name = 'im_livechat.chatbot.script_question_answer'
+    _description = 'Chatbot Script Question Answer'
```

Comment:
As other models: don't forget explicit order (and deterministic, always better, like ``sequence asc, id desc`` ).